backend/src/main/kotlin/com/github/quillraven/toilapp/dto/ToiletResultDto.kt

```diff
@@ -0,0 +1,8 @@
+package com.github.quillraven.toilapp.dto
+
+import com.github.quillraven.toilapp.model.Toilet
+
+data class ToiletResultDto(
```

Comment:
I think we can get rid of `ToiletResultDto`. There is a `@Transient` annotation which should tell a property to not get persisted. Therefore **distance** could become a property of Toilet. Does that make sense?